PersonalizeIDPretrained.yaml

```diff
@@ -0,0 +1,379 @@
+            Action: sts:AssumeRole
+      Path: "/"
+      Policies:
+        - PolicyName: !Sub "${AWS::StackName}-sagemaker-s3-poicy"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Action:
+                  - "s3:*"
+                  - "s3-object-lambda:*"
+                Resource: "*"
+                Effect: Allow
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/AmazonSageMakerFullAccess"
+        - "arn:aws:iam::aws:policy/service-role/AmazonPersonalizeFullAccess"
+        - "arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess"
+        - "arn:aws:iam::aws:policy/AWSStepFunctionsFullAccess"
+
+      # Tagging Ressources to leverage SSM's Application Manager.
+      # ---
+      Tags:
+        - Key: AppManagerCFNStackKey
+          Value: !Ref 'AWS::StackId'
+
+  # SageMaker notebook
+  NotebookInstance:
+    Type: "AWS::SageMaker::NotebookInstance"
+    Properties:
+      InstanceType: "ml.t3.medium"
+      NotebookInstanceName: !Ref NotebookName
+      RoleArn: !GetAtt SageMakerIamRole.Arn
+      VolumeSizeInGB: !Ref VolumeSize
+      LifecycleConfigName: !GetAtt AmazonPersonalizeMLOpsLifecycleConfig.NotebookInstanceLifecycleConfigName
+
+  # SSM Parameter with the s3 bucket for personalize
+  # ---
+  PersonalizeS3Bucket:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: /cloudformation/personalize-s3-bucket
+      Type: String
+      Value: !GetAtt MLOpsStack.Outputs.PersonalizeBucketName
+      Description: SSM Parameter containing the s3 bucket for Personalize Samples.
+      # Tagging Ressources to leverage SSM's Application Manager.
+      # ---
+      Tags:
+        Key: AppManagerCFNStackKey
+        Value: !Ref 'AWS::StackId'
+
+  # SSM Parameter with the IAM Role for personalize
+  # ---
+  PersonalizeIAMRole:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: /cloudformation/personalize-iam-role-arn
+      Type: String
+      Value: !GetAtt PersonalizeIamRole.Arn
+      Description: SSM Parameter containing the iam role arn for Personalize.
+      # Tagging Ressources to leverage SSM's Application Manager.
+      # ---
+      Tags:
+        Key: AppManagerCFNStackKey
+        Value: !Ref 'AWS::StackId'
+
+  AmazonPersonalizeMLOpsLifecycleConfig:
+    # The lifecycle configuration clones the github pository ad runs boostrapping code to deploy assets and pre-train models
+    # it uses the domain as a parameter in the file path to find the right deploy script
+
+    # TODO Change to actual immersion day repo
+    Type: "AWS::SageMaker::NotebookInstanceLifecycleConfig"
+    Properties:
+      OnStart:
+        - Content:
+            Fn::Base64:
+              !Sub |
+                #!/bin/bash
+                sudo -u ec2-user -i <<'EOF'
+                cd /home/ec2-user/SageMaker/
+                git clone --single-branch --branch immersion-day-refactor-eric https://github.com/ewbolme/amazon-personalize-immersion-day.git
```

Comment:
Should point to correct repo and not pull from a branch.

Reply:
will update this when its pulled into main repo - it needs to stay this way until the changes to the workshop are tested